extractor/extract.php

```diff
@@ -820,6 +821,44 @@ private function filterClassPhpDocs(Node\Stmt\ClassLike $class): Node\Stmt\Class
 		return $class;
 	}
 
+	private function filterFunctionPhpDocs(Node\Stmt\Function_ $function): Node\Stmt\Function_
+	{
+		$namespacedName = $function->namespacedName->toString();
```

Comment:
Comment with a link to a related discussion why precisely these functions need to be fixed would be appreciated!